BUILD/settings/any.dat

```diff
@@ -48,3 +48,4 @@ auto_forceFatLootToken	boolean	force grabbing the fat loot tokens from daily dun
 auto_pvpEnable	boolean	Break the hippy stone to unlock PvP?
 auto_towerBreak	string	Where should we break in the tower? (e.g. wall of skin, wall of bones, shadow, ns). Blank by default.
 auto_ignoreRestoreFailure	boolean	if true we will always ignore failure to restore MP or HP and just continue playing
+auto_useBatteries	boolean	If true will use harvest and use batteries during breakfast. If false will only harvest.
```

Comment:
I prefer to have no setting as well for the reasons you described. I put it in because I wasn't sure if some folks thought this would be too inefficient use of batteries and wanted to manually control. Suppose that goes against the entire point of being automatic though lol. I will remove the setting

Reply:
```suggestion
```